libcxx/include/mutex

```diff
@@ -419,8 +419,9 @@ inline _LIBCPP_HIDE_FROM_ABI void lock(_L0& __l0, _L1& __l1, _L2& __l2, _L3&...
 }
 
 #  endif // _LIBCPP_CXX03_LANG
+#endif   // !_LIBCPP_HAS_NO_THREADS
```

Comment:
Don’t you need to also remove the ifdef above here?

Reply:
I just moved the `#endif   // !_LIBCPP_HAS_NO_THREADS` from line 495 to line 422.

Reply:
Ack. Sorry, I misread the diff.